presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -176,13 +183,21 @@ private void writeShard(UUID shardUuid)
 
         if (isBackupAvailable()) {
             File backupFile = storageService.getBackupFile(shardUuid);
+            long start = System.nanoTime();
             storageService.createParents(backupFile);
+            stats.addCreateParentsTime(Duration.nanosSince(start));
+
             try {
+                start = System.nanoTime();
                 copyFile(storageFile.toPath(), backupFile.toPath());
             }
             catch (IOException e) {
                 throw new PrestoException(RAPTOR_ERROR, "Failed to create backup shard file", e);
             }
+            Duration duration = nanosSince(start);
+            DataSize size = new DataSize(storageFile.length(), BYTE);
+            DataSize rate = dataRate(size, duration).convertToMostSuccinctDataSize();
+            stats.addCopyShardDataRate(rate, size, duration);
```

Comment:
Oh, I wanted to add  a log line here. But I think I'll skip it and compute this in ``addCopyShardDataRate``